test/combinations.jl

```diff
@@ -449,3 +630,134 @@ end
+    @test_throws "not supported" volumeintegral(f, sphere)
+end
+
+@testitem "Meshes.Sphere 3D" setup=[Setup] begin
+    origin = Point(0, 0, 0)
+    sphere = Sphere(origin, 4.4)
+
+    f = p -> one(Float64)
+    fv(p) = fill(f(p), 3)
+
+    # Scalar integrand
+    sol = Meshes.measure(sphere)
+    @test integral(f, sphere, GaussLegendre(100)) ≈ sol
+    @test integral(f, sphere, GaussKronrod()) ≈ sol
+    @test integral(f, sphere, HAdaptiveCubature()) ≈ sol
+
+    # Vector integrand
+    vsol = fill(sol, 3)
+    @test integral(fv, sphere, GaussLegendre(100)) ≈ vsol
+    @test integral(fv, sphere, GaussKronrod()) ≈ vsol
+    @test integral(fv, sphere, HAdaptiveCubature()) ≈ vsol
+
+    # Integral aliases
+    @test_throws "not supported" lineintegral(f, sphere)
+    @test surfaceintegral(f, sphere) ≈ sol
+    @test_throws "not supported" volumeintegral(f, sphere)
+end
+
+@testitem "Meshes.Tetrahedron" setup=[Setup] begin
+    pt_n = Point(0, 1, 0)
+    pt_w = Point(-1, 0, 0)
+    pt_e = Point(1, 0, 0)
+    ẑ = Vec(0, 0, 1)
+    tetrahedron = Tetrahedron(pt_n, pt_w, pt_e, pt_n + ẑ)
+
+    f = p -> one(Float64)
+    fv(p) = fill(f(p), 3)
+
+    # Scalar integrand
+    sol = Meshes.measure(tetrahedron)
+    @test_throws "not supported" integral(f, tetrahedron, GaussLegendre(100))
+    @test integral(f, tetrahedron, GaussKronrod()) ≈ sol
+    @test_throws "not supported" integral(f, tetrahedron, HAdaptiveCubature())
+
+    # Vector integrand
+    vsol = fill(sol, 3)
+    @test_throws "not supported" integral(fv, tetrahedron, GaussLegendre(100))≈vsol
+    @test integral(fv, tetrahedron, GaussKronrod()) ≈ vsol
+    @test_throws "not supported" integral(fv, tetrahedron, HAdaptiveCubature())≈vsol
+
+    # Integral aliases
+    @test_throws "not supported" lineintegral(f, tetrahedron)
+    @test_throws "not supported" surfaceintegral(f, tetrahedron)
+    @test volumeintegral(f, tetrahedron, GaussKronrod()) ≈ sol
+end
+
+@testitem "Meshes.Torus" setup=[Setup] begin
+    origin = Point(0, 0, 0)
+    ẑ = Vec(0, 0, 1)
+    torus = Torus(origin, ẑ, 3.5, 1.25)
+
+    f = p -> one(Float64)
+    fv(p) = fill(f(p), 3)
+
+    # Scalar integrand
+    sol = Meshes.measure(torus)
+    @test integral(f, torus, GaussLegendre(100)) ≈ sol
+    @test integral(f, torus, GaussKronrod()) ≈ sol
+    @test integral(f, torus, HAdaptiveCubature()) ≈ sol
+
+    # Vector integrand
+    vsol = fill(sol, 3)
+    @test integral(fv, torus, GaussLegendre(100)) ≈ vsol
+    @test integral(fv, torus, GaussKronrod()) ≈ vsol
+    @test integral(fv, torus, HAdaptiveCubature()) ≈ vsol
+
+    # Integral aliases
+    @test_throws "not supported" lineintegral(f, torus)
+    @test surfaceintegral(f, torus) ≈ sol
+    @test_throws "not supported" volumeintegral(f, torus)
+end
+
+@testitem "Meshes.Triangle" setup=[Setup] begin
+    pt_n = Point(0, 1, 0)
+    pt_w = Point(-1, 0, 0)
+    pt_e = Point(1, 0, 0)
+    triangle = Ngon(pt_e, pt_n, pt_w)
```

Comment:
```suggestion
    triangle = Triangle(pt_e, pt_n, pt_w)
```
To clarify that this really is a triangle.